ups/obs_subaru.table

```diff
@@ -1,19 +1,13 @@
-setupRequired(daf_persistence >= 5.0.0.1)
-setupRequired(daf_butlerUtils >= 3.0.3)
-setupRequired(skypix >= 3.0.3)
+setupRequired(daf_persistence)
+setupRequired(daf_butlerUtils)
+setupRequired(skypix)
 setupRequired(pipe_tasks)
 setupRequired(meas_astrom)
 setupRequired(pyfits)
 
-setupOptional(psycopg2)
-setupOptional(meas_extensions_multiShapelet)
-setupOptional(meas_multifit)
+setupOptional(meas_modelfit)
 setupOptional(meas_extensions_psfex)
 
-setupOptional(suprime_data)
-
-setupOptional(testdata_subaru)
```

Comment:
It's a fair point. My initial reaction was to simply skip the tests on the assumption that they'll be covered [elsewhere](https://github.com/lsst/ci_hsc) pretty soon. On reflection, though, it's probably worth spending a few minutes putting together a minimal test data package and doing some bare-bones testing à la `testdata_decam`. Watch this space.